tests/integration/test_task_worker.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+from cabbage.tasks import TaskManager
+from cabbage.task_worker import Worker
+
+
+@pytest.fixture()
+def results():
+    return {"sums": [], "products": []}
+
+
+@pytest.fixture()
+def task_manager(connection, results):
+    manager = TaskManager(connection)
+
+    @manager.task(queue="sum_queue")
+    def sum_task(_, a, b):
+        results["sums"].append(a + b)
+
+    @manager.task(queue="sum_queue")
+    def stop_task(_):
+        raise KeyboardInterrupt()
+
+    sum_task.defer(a=1, b=2)
+    stop_task.defer()
+
+    return manager
+
+
+@pytest.fixture()
+def worker(task_manager):
+    return Worker(task_manager, "sum_queue")
+
+
+def test_run(worker, results):
+    worker.run()
```

Comment:
Wow, I like the way you're leaving 100% of the test setup to fixtures. Does it scale properly when you have more tests ?